doc/adr/0005-bibliotheque-graphes.md

```diff
@@ -0,0 +1,73 @@
+
+5. Documentation exhaustive
+
+    et souvent interactive de [toutes les options](https://echarts.apache.org/en/option.html)
+
+    Offre aussi des raccourcis classés [par composants](https://echarts.apache.org/en/cheat-sheet.html) facilitant la découvrabilité.
+
+6. Projet pérenne
+
+    - Soutenu par la fondation Apache
+    - Existe depuis 2018
+    - 60k+ stars GH
+    - Maintenu par de nombreux contributeurs
+    - Non spécifique à React
+
+7. Autres fonctionnalités
+
+[Support SSR](https://echarts.apache.org/handbook/en/how-to/cross-platform/server/#server-side-rendering)
+
+[Gestion des données intégrées](https://echarts.apache.org/handbook/en/concepts/dataset)
+
+### Négatives
+
+1. Nouvelle API à apprendre
+
+2. Remplacer les graphes existants
+
+Et notamment ceux des indicateurs pour afficher correctement les valeurs négatives
+
+## Alternatives considérées
+
+Les autres bibliothèques du marché (recharts, victory, etc.) sont souvent trop liées à React, moins riches fonctionnellement et avec un système d'exemples/playground moins pratique.
```

Comment:
Pour le coup, je reste un peu plus sur ma faim sur cette partie là. De mon point de vue, le lien avec React n'est pas trop grave étant donné qu'on ne fait que du React. Par ailleurs, ce n'est pas le cas de chartjs qui est également très largement utilisé. Donc je ne mettrais pas ce sujet en premier.
Par contre la richesse de rendue est effectivement un très bon point pour ne pas être limité par la suite. Et le SSR en est également un autre.
Le système d'exemple plus du mal à me faire un avis car je ne l'ai pas manipulé et la doc de chartjs est très bien faite également.
Peut être inclure un ou deux articles plus détaillés de comparaison ? J'ai trouvé celui là qui m'a éclairé: https://theaverageprogrammer.hashnode.dev/choosing-the-right-charting-library-for-your-nextjs-dashboard.
S'il te parle également on peut l'ajouter.

Reply:
Merci pour les retours. J'ai un peu complété.